### ISSUE TITLE
Swagger UI showing error for JSON generated when using nullable

### ISSUE THREAD

**Issue by bhrigushr** [nestjs/swagger#1439]
## Current behavior

Swagger JSON generated is having an error when checked in https://editor.swagger.io/ when using `nullable:true` field

<img width="1792" alt="Screenshot 2021-07-12 at 1 02 11 PM" src="https://user-images.githubusercontent.com/56547349/125248335-904f5700-e311-11eb-9ed8-2af2b97b88f4.png">

## Expected behavior

There should be no error in Swagger JSON generated

What I observed is that `nullable` is not placed inside `schema` object which is causing this error

Currently
```@javascript
{
            "name": "offset",
            "required": false,
            "in": "query",
            "description": "The offset is the position in the dataset of a particular record.",
            "nullable": true,
            "schema": {
              "format": "integer",
              "type": "number"
            }
}
```

Expected
```@javascript
{
            "name": "offset",
            "required": false,
            "in": "query",
            "description": "The offset is the position in the dataset of a particular record.",
            "schema": {
              "format": "integer",
              "type": "number",
              "nullable": true,
            }
}
```

**Input Code**

```@javascript
export class TestDto {
  @ApiProperty({
    description: 'The offset is the position in the dataset of a particular record.',
    required: false,
    nullable: true,
    format: 'integer',
  })
  offset?: number;

  @ApiProperty({
    description: 'This is the size of the results set you would like.',
    required: false,
    nullable: true,
    format: 'integer',
    example: 100,
  })
  limit?: number;
}


@Get('/test')
  @ApiOperation({
    description: "Test API",
  })
  public async get(
    @Res() res,
    @Query() query: TestDto,
    @Request() req,
  ): Promise<void> {
  ...
  ...
  }
```

## Environment

<pre><code>
@nestjs/core: 7.6.17
@nestjs/swagger: 4.8.0

 
For Tooling issues:
- Node version: v12.18.4  
- Platform:  Mac

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Please provide a minimum reproduction repository.

**Comment by bhrigushr** [comment]
@kamilmysliwiec reproduction [repository](https://github.com/bhrigushr/nestjs-swagger)

**Comment by kamilmysliwiec** [comment]
Fixed in 5.0.4
